python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -1218,16 +1262,12 @@ def convert_controlnet(pipe, args):
         del coreml_controlnet
         gc.collect()
 
-def get_pipeline(args):
-    if 'xl' in args.model_version:
-        pipe = StableDiffusionXLPipeline.from_pretrained(args.model_version,
-                                                       torch_dtype=torch.float16,
-                                                       variant="fp16",
-                                                       use_safetensors=True,
-                                                       use_auth_token=True)
-    else:
-        pipe = StableDiffusionPipeline.from_pretrained(args.model_version,
-                                                       use_auth_token=True)
+def get_pipeline(model_version):
```

Comment:
This breaks the use of `get_pipeline` here: https://github.com/apple/ml-stable-diffusion/blob/c5063225aa1473522116425268bb7285e6020f9b/python_coreml_stable_diffusion/mixed_bit_compression_apply.py#L51 and here: https://github.com/apple/ml-stable-diffusion/blob/c5063225aa1473522116425268bb7285e6020f9b/python_coreml_stable_diffusion/mixed_bit_compression_pre_analysis.py#L407

Maybe fix those, or go back to passing `args` for compatibility?

Reply:
Good catch, thanks for testing this out - missed these other usages. Just updated it again so that the interface is the same.